src/index.scss

```
@@ -319,9 +319,13 @@
   max-width: 640px;
   margin: 0 auto;
 
+  @media (max-width: -1 + map-get($grid-breakpoints, "sm")) {
+    flex-direction: column;
+    gap: 16px;
```

[nit]: Try to use this

```suggestion
    gap: $spacer;
```

---

Changed, thanks